rust-version

```diff
@@ -1 +1 @@
-506e75cbf8cb5305e49a41326307004ca3976029
+e1ff91f439bc09f566da211c6449821b4e949279
```

Comment:
This file is missing a trailing newline, causing the `./rustup-toolchain` script always changes this file to insert a newline here.